### PR TITLE
fix: Change position of create button

### PR DESCRIPTION
**Pull request by samuveth**
### Issues

Fixes design changes on Figma 

### Changes 

1. Move the "New proposal" button to the top out of the sidebar

### How to test

1. Go to any space proposals page

Screenshot 2023-08-02 at 10 09 02 PM
Screenshot 2023-08-02 at 10 09 10 PM
Screenshot 2023-08-02 at 10 09 19 PM
Screenshot 2023-08-02 at 10 09 28 PM

### Self-review checklist
- [x] I have performed a full self-review of my changes
- [x] I have tested my changes on a preview deployment
- [x] I have tested my changes on different screen sizes (sm, md)
- [x] I have tested my changes on a custom domain

### CODE REVIEW

File: src/components/SpaceSidebarHeader.vue [hunk 1]
```diff
@@ -30,19 +30,12 @@ watchEffect(() => {
...
-        :space="space"
-        symbol-index="space"
-        size="80"
-        class="lg:my-3"
-      />
-      <div
-        class="mx-3 flex flex-col justify-center truncate text-left lg:block lg:text-center"
-      >
-        <h3 class="mb-[2px] flex items-center lg:justify-center">
+    <div class="">
```
Comment (wa0x6e) [inline src/components/SpaceSidebarHeader.vue:35]: Why the empty `class` ?

**Review by wa0x6e — CHANGES_REQUESTED** [review]

File: src/components/SpaceSidebarHeader.vue (hunk 1 above)
Comment (samuveth) [inline src/components/SpaceSidebarHeader.vue:35]:
```suggestion
    <div>
```

Comment (samuveth) [inline src/components/SpaceSidebarHeader.vue]:
```suggestion
    <div>
```

**Review by wa0x6e — APPROVED** [review]
utAck

**Review by wa0x6e — APPROVED** [review]
utAck

**Pull request merged**